Doc/library/logging.rst

```diff
@@ -1157,8 +1158,8 @@ functions.
 
 .. function:: getLogger(name=None)
 
-   Return a logger with the specified name or, if name is ``None``, return a
-   logger which is the root logger of the hierarchy. If specified, the name is
+   Return a logger with the specified name or, if name is ``None``, return the
+   root logger of the hierarchy (equivalent to passing ``name='root'``). If specified, the name is
```

Comment:
I don't especially want to encourage the idiom where you fetch the root logger using `name='root'`. For example, code Python 2 doesn't work like this, and while Python 2.x is EOL, there are still environments where it is used (and there might be other environments like Jython where this isn't the case). So I'd prefer to suggest `logging.getLogger()` as the default way of getting the root logger, while perhaps noting that `logging.getLogger('')` or `logging.getLogger(None)` also work for this.